python/sglang/srt/utils.py

```diff
@@ -98,6 +98,38 @@ def alloc_usable_network_port(num, used_list=()):
                 return port_list
     return None
 
+def check_port(port):
```

Comment:
```suggestion

def check_port(port):
```

Two empty lines between functions